spec/design/skel/AARCH64/ArchInvocationLabels_H.thy

```diff
@@ -21,9 +21,8 @@ text \<open>
 
 end
 
-context begin interpretation Arch .
-requalify_types arch_invocation_label
-end
+arch_requalify_types (H)
+  arch_invocation_label
```

Comment:
Out of curiosity since you didn't mention it in the commit or leave a comment, why can't this one be moved to a generic theory?

Reply:
Can't be moved due to enum instantiations (enum, enum_alt, enumeration_both) right after it, which can't be done inside a locale. Want me to add a comment?

Reply:
Yeah, I think a comment would be helpful.

Reply:
going with:
```
(* not possible to move this requalification to generic, since enum instance proofs must
   be done outside of Arch locale *)
```
aside: it's really surprising how we set up the Arch locale, and then the Haskell translator immediately ends it to do the enum instance proofs outside the locale using `qualify/end_qualify`, after which it restarts the Arch locale, and that one's ended by the `end` you see in the skeleton file.